src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -390,6 +390,14 @@ ServerHandlerImp::onStopped (Server&)
     stopped();
 }
 
+static
+void
+extract_params(Json::Value& jv, Json::Value const& params)
```

Comment:
Good catch.  It originally was used, but no longer is after a rewrite.  Forgot to take it back out, thanks.